test/glyphs.js

```diff
@@ -155,56 +155,101 @@ describe('glyphs', function() {
     });
   });
 
-  describe('WOFF glyphs', function() {
-    let font = fontkit.openSync(__dirname + '/data/SourceSansPro/SourceSansPro-Regular.woff');
+  describe('WOFF ttf glyphs', function() {
+    let font = fontkit.openSync(__dirname + '/data/SourceSansPro/SourceSansPro-Regular.ttf.woff');
+    let glyph = font.glyphsForString('D')[0];
+
+    it('should get the glyph name', function() {
+      return assert.equal(glyph.name, 'D');
+    });
 
     it('should get a TTFGlyph', function() {
-      let glyph = font.glyphsForString('T')[0];
       return assert.equal(glyph.constructor.name, 'TTFGlyph');
     });
 
-    it('should get a path for the glyph', function() {
-      let glyph = font.glyphsForString('T')[0];
-      return assert.equal(glyph.path.toSVG(), 'M226 586L28 586L28 656L508 656L508 586L310 586L310 0L226 0Z');
+    it('should get a quadratic path for the glyph', function() {
+      return assert.equal(glyph.path.toSVG(), 'M90 0L90 656L254 656Q406 656 485 571.5Q564 487 564 331Q564 174 485.5 87Q407 0 258 0ZM173 68L248 68Q363 68 420.5 137.5Q478 207 478 331Q478 455 420.5 521.5Q363 588 248 588L173 588Z');
     });
+  });
 
-    it('should get the glyph name', function() {
-      let glyph = font.glyphsForString('T')[0];
-      return assert.equal(glyph.name, 'T');
-    });
+  describe('WOFF otf glyphs', function() {
+      let font = fontkit.openSync(__dirname + '/data/SourceSansPro/SourceSansPro-Regular.otf.woff');
```

Comment:
I don't think there's a prettifier running for the source at the moment, but it looks like this got indented twice.